### PR TITLE
model implementation

### PR DESCRIPTION
**Pull request by jaynopponep**
- found pre-trained model collected and labeled by _david-lee-d0rhs_ on RoboFlow (authored by @jaynopponep) 
- used yolov11 to train on pretrained model to give high accuracy sign interpretation (authored by @jaynopponep) 
- implemented mediapipe use to assist in prediction (authored by @jaynopponep)
- fixed bounding boxes in order to identify more crucial parts of hand signing (wrist, tip of fingers, etc) (authored by @Claudio-Perinuzzi) 
- wrote functionality for taking max prediction within 5 seconds of signing to get the best prediction (authored by @Claudio-Perinuzzi)

### CODE REVIEW

**Comment by jaynopponep** [comment]
Imma look at the clutter in this branch and then merge to get the ball rolling since there's nothing in main
